app/routes/__main/subscription/__main/activeSubscription/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const activeSubscription = () => (
+   <section className="screen create tabContainer">
+      <div>
+         <h1 style={{color: "black"}}>hello</h1>
+      </div>
```

Comment:
This should be another component imported from `app/components/ActiveSubscription.tsx`